keyboards/aki27/cocot46plus/cocot46plus.h

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022 aki27
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "quantum.h"
+
+typedef union {
+    uint32_t raw;
+    struct {
+        uint8_t cpi_idx;
+        uint8_t scrl_div;
+        uint8_t rotation_angle;
+        int8_t scrl_inv;
+        bool scrl_mode;
+        report_mouse_t last_mouse;
+    };
+} cocot_config_t;
```

Comment:
The inner struct is obviously larger than `uint32_t`, so only the first 4 fields get saved in the EEPROM, even if the appearance of this union suggests otherwise. To avoid confusion, you should split this into separate data items for the persistent and runtime-only data.

Reply:
I'm adding code commits to clarify that `scrl_mode` and `last_mouse` are not saved to EEPROM